modules/metastorage/src/main/java/org/apache/ignite/internal/metastorage/MetaStorageManager.java

```diff
@@ -874,28 +915,31 @@ private long appliedRevision() {
      *
      * @return Ignite UUID of new consolidated watch.
      */
-    private CompletableFuture<Optional<IgniteUuid>> updateWatches() {
-        long revision = appliedRevision() + 1;
+    private CompletableFuture<IgniteUuid> updateWatches() {
```

Comment:
I think it's worth mentioning in a comment that this method MUST be called under `synchronized`